src/proxy-animation.js

```diff
@@ -1623,8 +1624,7 @@ function parseTimeRange(value) {
   });
 
   if (names.length > 2 || offsets.length > 2 || offsets.length == 1) {
-    // Reject value and use default.
-    return timeRange;
+    throw new Error("Invalid time range");
```

Comment:
nit: This should be a TypeError, as we are unable to parse the supplied arguments as the expected type.

Reply:
done